htlcswitch/switch_test.go

```diff
@@ -1872,3 +1877,24 @@ func TestMultiHopPaymentForwardingEvents(t *testing.T) {
 		}
 	}
 }
+
+// TestLogTicker verifies that the logTicker ticked by waiting 11 seconds after the start of switch
+// without this the code handling the logTicker will not be called and there will be a drop in coverage stats (build fail)
+func TestLogTicker(t *testing.T) {
```

Comment:
The coverage stats have been deemed a bit unreliable for some time now, so not sure if this is something we should worry about.